packages/providers/src/utils/fee.test.ts

```diff
@@ -48,21 +59,21 @@ describe(__filename, () => {
         },
       ];
 
-      const result = calculateTransactionFeeForScript({
+      const result = calculateTransactionFee({
         receipts,
         gasPrice,
         gasPriceFactor,
         margin,
+        transactionBytes,
+        transactionWitnesses,
+        gasPerByte,
       });
 
-      const expectedFee = bn(
-        Math.ceil(gasUsed.toNumber() / gasPriceFactor.toNumber()) * gasPrice.toNumber()
-      );
-
+      const expectedFee = 1200;
       const expectedGasUsed = gasUsed;
 
-      expect(result.fee).toStrictEqual(expectedFee);
-      expect(result.gasUsed).toStrictEqual(expectedGasUsed);
+      expect(result.fee.toNumber()).toStrictEqual(expectedFee);
```

Comment:
@camsjams this is the final expected fee value. The value is a lot higher because I've updated the mocks and I am using real transaction bytes and real transaction witnesses as well